src/NuGet.Clients/NuGet.SolutionRestoreManager/Resources.resx

```diff
@@ -140,6 +140,9 @@
 
 Missing packages: {0}</value>
   </data>
+  <data name="PackageRefNotRestoredBecauseOfNoConsent" xml:space="preserve">
+    <value>NuGet package couldn't be restored because consent has not been granted.To give consent, open the Visual Studio Options dialog, click on the Package Manager node and check 'Allow NuGet to download missing packages during build.' You can also give consent by setting the environment variable 'EnableNuGetPackageRestore' to 'true'.</value>
```

Comment:
NuGet package**s**. 
granted.[space]To give consent

Maybe this message should be equivalent to the other one?
One or more NuGet packages need... etc?